pom.xml

```diff
@@ -72,53 +68,45 @@
   </scm>
 
   <properties>
-    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <project.build.outputEncoding>UTF-8</project.build.outputEncoding>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <maven.findbugs.failure.strict>true</maven.findbugs.failure.strict>
+    <jenkins.version>1.554.1</jenkins.version>
+    <!-- TODO check if we'd better add the PI. -->
+    <jelly.requirePI>false</jelly.requirePI>
```

Comment:
Yes you should add the PI. The rule is that all `*.jelly` should explicitly say they escape output by default; where you have `something.jelly`

```xml
${something}
```

and `something` might _legitimately_ include HTML tags you want to render raw without risk of XSS (for example because it is coming from `something.properties`, or is a Java getter where the value is known not be “tainted” by user input), you can use

```xml
<j:out value="${something}"/>
```

By updating to a new parent POM which uses a new version of the HPI plugin that includes this `InjectedTest`, you confirm that you have reviewed all your views for XSS vulnerabilities.